CHANGELOG.md

```diff
@@ -1,6 +1,25 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## Wazuh Puppet v4.0.4
+
+### Added
+
+- Update to Wazuh version 4.0.4
```

Comment:
Should we add a link to core changelog here as well?

Reply:
Done on 6291ae4f54703523c145b2fbd2df52ab2de1f97f, thanks!